blazar/plugins/oshosts/host_plugin.py

```diff
@@ -73,6 +75,7 @@
 
 
 before_end_options = ['', 'snapshot', 'default', 'email']
+on_start_options = ['', 'default', 'orchestration']
```

Comment:
This `orchestration` value appears to be unused, but I think it's a good idea. I would go a bit further though - what should this API look like? Currently you have it as `on_start=ee9cc0b2-5ce7-4652-9e06-8f10d9bed0c2` (for example). This makes it a bit hard to extend `on_start` to do more things in the future. 

Perhaps you can express it as `on_start=launch_stack:ee9cc0b2-5ce7-4652-9e06-8f10d9bed0c2` or `on_start=orchestration:ee9cc0b2-5ce7-4652-9e06-8f10d9bed0c2` or similar. I don't see that Blazar has much support for passing options for these params. 

Another possibility is doing it similar to reservations, so `on_start=launch_stack,stack=ee9cc0b2-5ce7-4652-9e06-8f10d9bed0c2` - using commas to separate the pseudo-params. What do you think would be best? We would have to look at increasing the column length of this field if we did this.

Reply:
I updated to 'orchestration:<uuid>'